patches/html-entities+2.5.2.patch

```diff
@@ -0,0 +1,8 @@
+diff --git a/node_modules/html-entities/lib/index.js b/node_modules/html-entities/lib/index.js
```

Comment:
I guess this will also be necessary in apps using the lib?

Reply:
That's correct, we'll need to patch `html-entities` inside E/App as well.